Source/WebCore/page/Page.h

```diff
@@ -1586,6 +1588,7 @@ class Page : public RefCountedAndCanMakeWeakPtr<Page>, public Supplementable<Pag
     bool m_mediaPlaybackIsSuspended { false };
     bool m_mediaBufferingIsSuspended { false };
     bool m_hasResourceLoadClient { false };
+    bool m_hasPaintedHDRContent { false };
```

Comment:
Page is the wrong place to store this. It should be on Document or LocalFrameView.